### PR TITLE
Stop using IISExpress .config templates

### PR DESCRIPTION
**Pull request by pakrym**
Fixes test failures

### CODE REVIEW

File: test/ServerComparison.FunctionalTests/ResponseCompressionTests.cs [hunk 1]
```diff
@@ -96,13 +98,34 @@ public Task ResponseCompression_AppAndHostCompression(TestVariant variant)
...
+                        : Helpers.GetNginxConfigContent("NoCompression.conf");
                 }
-                else
+                else if (variant.Server == ServerType.IISExpress)
                 {
-                    deploymentParameters.ServerConfigTemplateContent = Helpers.GetConfigContent(variant.Server, "NoCompression.config", "NoCompression.conf");
+                    var iisDeploymentParameters = new IISDeploymentParameters(deploymentParameters);
+                    iisDeploymentParameters.ServerConfigActionList.Add(
+                        (element, _) => {
+                            var compressionElement = element
```
Comment (Tratcher) [inline test/ServerComparison.FunctionalTests/ResponseCompressionTests.cs]: How does this adjust for hostCompression?

File: test/ServerComparison.FunctionalTests/ResponseCompressionTests.cs [hunk 2]
```diff
@@ -96,13 +98,34 @@ public Task ResponseCompression_AppAndHostCompression(TestVariant variant)
...
+                                .RequiredElement("system.webServer")
+                                .RequiredElement("httpCompression");
+
+                            compressionElement
+                                .RequiredElement("dynamicTypes")
+                                .Elements()
+                                .SkipLast(1)
```
Comment (Tratcher) [inline test/ServerComparison.FunctionalTests/ResponseCompressionTests.cs]: ??

**Review by Tratcher — CHANGES_REQUESTED** [review]
Tests are still failing.

**Comment by pakrym** [comment]
> Tests are still failing.

Not anymore.

File: test/ServerComparison.FunctionalTests/ResponseCompressionTests.cs (hunk 2 above)
Comment (pakrym) [reply test/ServerComparison.FunctionalTests/ResponseCompressionTests.cs]: This is exactly the change NoCompression.config did - removing all but last element from `dynamicTypes` and `staticTypes`

Comment (pakrym) [reply test/ServerComparison.FunctionalTests/ResponseCompressionTests.cs]: image

Comment (Tratcher) [reply test/ServerComparison.FunctionalTests/ResponseCompressionTests.cs]: Deserves a comment about how that last element disables compression

Comment (pakrym) [reply test/ServerComparison.FunctionalTests/ResponseCompressionTests.cs]: Done

**Review by Tratcher — APPROVED** [review]

**Pull request merged**